fuel/app/themes/default/partials/my_widgets.php

```diff
@@ -82,10 +82,8 @@
 									<span class="owner">Full</span>
 									<span class="undo">Removed <a href="#">Undo</a></span>
 
-									<select ng-disabled="selected.shareable==false" tabindex="0" id="perm" class="perm" ng-model="collaborator.access" ng-change="checkForWarning(collaborator)">
-										<option value={{ACCESS.FULL}} ng-selected="collaborator.access == ACCESS.FULL" >Full</option>
-										<option value={{ACCESS.VISIBLE}} ng-selected="collaborator.access == ACCESS.VISIBLE" >View Scores</option>
-									</select>
+									<select ng-disabled="selected.shareable==false" tabindex="0" id="perm" class="perm" ng-model="collaborator.access" ng-change="checkForWarning(collaborator)" ng-options="accessLevel.value as accessLevel.text for accessLevel in accessLevels">
```

Comment:
The `ng-options` directive seems like maybe it's too restrictive for our purposes here. Would recommend using an an `ng-repeat` and defining the options separately.

```suggestion
<select ng-disabled="selected.shareable==false"
	tabindex="0"
	id="perm"
	class="perm"
	ng-model="collaborator.access"
	ng-change="checkForWarning(collaborator)">
	<option ng-repeat="level in accessLevels"
		value="{{level.value}}">
		{{level.text}}
	</option>
</select>
```

This probably won't solve the original problem, which was that newly added collaborators' access level was defaulted to an empty third option. I believe that's a problem that will need to be solved in the client-side code.